src/jdk.jlink/share/classes/jdk/tools/jlink/internal/Platform.java

```diff
@@ -128,6 +180,16 @@ private static Architecture toArch(String archName) {
             case "amd64", "x86_64" -> Architecture.x64;
             case "arm"             -> Architecture.ARM;
             case "aarch64"         -> Architecture.AARCH64;
+            case "loongarch64"     -> Architecture.LOONGARCH64;
+            case "ppc"             -> Architecture.PPC;
+            case "ppc64"           -> Architecture.PPC64;
+            case "ppc64le"         -> Architecture.PPC64LE;
+            case "riscv32"         -> Architecture.RISCV32;
+            case "riscv64"         -> Architecture.RISCV64;
+            case "s390"            -> Architecture.s390;
+            case "s390x"           -> Architecture.s390x;
+            case "sparc"           -> Architecture.SPARC;
+            case "sparcv9"         -> Architecture.SPARCv9;
```

Comment:
I've now updated the PR to trim down the target platforms to match what we have in `target.properties`. Additionally, `is64Bit` has been enhanced too.

I would like to note that, in an intermediate version of this PR, I made `is64Bit` return true for `arm` architecture. I did so because from the materials I found, ARM supports both 32 bit and 64 bit architectures, but the recent versions are 64 bit. So I thought it would be appropriate to consider it 64 bit.

However, I then looked up the `platform.m4` build file and it marks `arm` architecture as 32 bit https://github.com/openjdk/jdk/blob/master/make/autoconf/platform.m4#L57. So I've updated the PR to return false from `is64Bit` from `arm` arch.